README.md

```diff
@@ -5,23 +5,251 @@
 
 # Istio Operator
 
```

Comment:
you need a `##` here.  not sure what the `##` should be - possibly `## Introduction`

the `#Istio Operator` won't be displayed I think, but instead will be used as the title of the document.